src/python/pants/backend/python/target_types.py

```diff
@@ -551,6 +556,47 @@ def validate(self) -> None:
             )
 
 
+class PexEntryPointsField(StringSequenceField, AsyncFieldMixin):
+    alias = "entry_points"
+    default = None
+    help = (
+        # @TODO: Update
```

Comment:
Opening issue for TODO

Reply:
Would it help for me to send over some suggested language?

Reply:
Sure thing. I just wasn't feeling particularly English-y when I was coding :sweat_smile:

Reply:
> The entry points for each binary, i.e. what gets run when when executing `./my_app.pex`.
>
> Use a file name, relative to the BUILD file, like `app.py`. You can also set the function to run, like `app.py:func`. Pants will convert these file names into well-formed entry points, like `app.py:func` into `path.to.app:func`.
>
> If you want the entry point to be for a third-party dependency or to use a console script, use the `pex_binary` target directly.